lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -142,6 +144,7 @@ def main():
 
                 h5in.copy_node(k, g, overwrite=True)
 
+    write_dl2_dataframe(dl2.astype(float), output_file)
```

Comment:
Why convert dl2 as float?

Reply:
Yes, this does not make sense. There are integer columns in there and they should stay integers.

Reply:
sorry, just copied the line from the old version of the file